src/appshell/view/internal/splashscreen/splashscreen.h

```diff
@@ -33,7 +33,7 @@ class SplashScreen : public QWidget
 public:
     enum SplashScreenType {
         Default,
-        ForNewInstance
+        ForSecondaryInstance
```

Comment:
why did you rename it?
for the third and next will be secondary?

Reply:
I renamed it to make clear that this is only for non-main instances, not launched by the user but by some other instance. In this case, `secondary` means not "second" but "non-main".

Reply:
I would recommend returning it the way it was.
We also use NewInstance in NewInstanceLoadingScreenView